x-pack/test_serverless/api_integration/test_suites/observability/fleet.ts

```diff
@@ -16,7 +16,7 @@ export default function ({ getService }: FtrProviderContext) {
     it('rejects request to create a new fleet server hosts', async () => {
       const { body, status } = await supertest
         .post('/api/fleet/fleet_server_hosts')
-        .set(svlCommonApi.getCommonRequestHeader())
+        .set(svlCommonApi.getInternalRequestHeader())
```

Comment:
the API's public, not internal. Fleet has an open issue to change to 'public' again: https://github.com/elastic/ingest-dev/issues/1921

Reply:
This means until the linked issue is resolved, this API would be treated as internal ?